subsys/bluetooth/audio/ascs.c

```diff
@@ -354,6 +356,29 @@ static void ase_exit_state_streaming(struct bt_ascs_ase *ase)
 	} else {
 		LOG_WRN("No callback for stopped set");
 	}
+	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED) {
+		if (ops != NULL && ops->disabled != NULL) {
+			ops->disabled(stream);
+		} else {
+			LOG_WRN("No callback for disabled set");
+		}
+	}
```

Comment:
Since the source ASE has a different state machine, it is already handled there. This behavior should only apply for sink ASEs as per the reported issue. 

This already work implicitly as a source ASE will never go from Streaming -> QoS configured, but I think adding a check for sink ASE would be nice from a readability standpoint (or at the very least a comment).